### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by pixie79**
Allow template overrides,
Fix up error summary to include links

### CODE REVIEW

File: app/pages/task/form/components/TaskForm.jsx [hunk 1]
```diff
@@ -1,72 +1,111 @@
...
+    },
+  };
 
-        const variableInput = form.components.find(c => c.key === 'submitVariableName');
-        const variableName = variableInput ? variableInput.defaultValue : form.name;
+  render() {
+    const { onCustomEvent, variables, task, onSubmitTaskForm, formReference, form } = this.props;
+    const formVariableSubmissionName = `${form.name}::submissionData`;
+    const submissionData = variables['submissionData'] ? variables['submissionData'] : (variables[formVariableSubmissionName]
+      ? variables[formVariableSubmissionName] : {});
```
Comment (p3dr0migue1) [inline app/pages/task/form/components/TaskForm.jsx]: can we change this to:
```js
let submissionData = {};

if (variables['submissionData']) {
  submissionData = variables['submissionData']
} else if (variables[formVariableSubmissionName]) {
  submissionData = variables[formVariableSubmissionName];
}
```

its slightly less confusing.

File: public/styles/app.scss [hunk 2]
```diff
@@ -39,6 +39,11 @@ iframe {
   }
 }
 
+.govuk-accordion-no-bottom {
+  border-bottom: none !important;
```
Comment (p3dr0migue1) [inline public/styles/app.scss]: do we really need `!important` here? couldn't we just call the custom parent `id` or `class` in order to override the border?

Comment (pixie79) [reply public/styles/app.scss]: no, as we only use the parent in its own right as well, this allows us to override some of the time but not all

File: app/pages/task/form/components/TaskForm.jsx (hunk 1 above)
Comment (pixie79) [reply app/pages/task/form/components/TaskForm.jsx]: that does not look the same intent to me ?

Comment (p3dr0migue1) [reply app/pages/task/form/components/TaskForm.jsx]: well this `(variables[formVariableSubmissionName] ? variables[formVariableSubmissionName] : {})` only evaluated if there is no `variables['submissionData']`

**Pull request merged**